data/json/items/ammo/300.json

```diff
@@ -0,0 +1,40 @@
+[
+  {
+    "id": "300_winmag",
+    "type": "AMMO",
+    "name": ".300 Winchester Magnum",
+    "name_plural": ".300 Winchester Magnum",
+    "description": ".300 Winchester Magnum rounds with 220gr JHP bullets.  The 300WM round is an extremely powerful and accurate rifle round introduced in 1963.  It has proven popular with hunters and snipers, although it is not as common as .308 or .30-06.",
+    "weight": 28,
+    "volume": 1,
```

Comment:
It's arbitrary but in proportion to other cartridges. If someone wants to tabulate the correct weights and convert them that's fine but preferably not piecemeal.